sig-security-tooling/cve-feed/hack/fetch-official-cve-feed.py

```diff
@@ -50,7 +50,8 @@
     'items': None,
 }
 # format the timestamp the same way as GitHub RFC 3339 timestamps, with only seconds and not milli and microseconds.
-root_kubernetes_io = {'updated_at': datetime.utcnow().isoformat(sep='T', timespec='seconds') + 'Z'}
+root_kubernetes_io = {'prow_job': 'https://testgrid.k8s.io/sig-security-cve-feed#auto-refreshing-official-cve-feed',
```

Comment:
Can we rename this to `feed_refresh_job` since end users won't most likely know what prow is in context of K8s?

Reply:
yeah sure